src/reducers/index.js

```diff
@@ -2,9 +2,13 @@ import { combineReducers } from 'redux';
 import { reducer as formReducer } from 'redux-form';
 import authReducer from './auth';
 import signupReducers from './signupReducers';
+import createArticleReducer from './createArticleReducer';
+import viewArticleReducer from './viewArticleReducer';
 
 export default combineReducers({
   auth: authReducer,
   form: formReducer,
   signup: signupReducers,
+  articleCreate: createArticleReducer,
+  viewArticle: viewArticleReducer,
```

Comment:
Good job @oluwajuwon on delivering this task. I know you must have learned a lot while you were at it. However, though, a few things need to be adjusted. It's important we pay close attention to how we design our application store (Redux store). 

As each property passed to `combineReducers` becomes a state for our application to use, we want to make sure those names are carefully chosen and the reducers operating on those state are well defined. 

The `createArticleReducer` and `viewArticleReducer` could be combined into a single reducer operating on a single state (`article`). I think it would be nice to have an _articleReducer_ file that knows how to manipulate the (`article`) state of our application.

Please let me know if you need further clarifications. Weldone.

Reply:
Thanks @codeshifu, I actually learnt alot 😄.
About the reducers, I didn't think of it that way, but now that I've put it more into perspective and reflected on it, I totally understand it now. I'll implement that and get back to you. Thanks 👍